geopandas/io/tests/test_parquet.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import absolute_import
+import os
+import pytest
+from pandas import DataFrame
+import numpy as np
+
+from geopandas import GeoDataFrame, GeoSeries, read_file, read_parquet
+from geopandas.array import to_wkb
+from geopandas.datasets import get_path
+from geopandas.io.parquet import (
+    _encode_metadata,
+    _decode_metadata,
+    _validate_dataframe,
+)
+from geopandas.testing import assert_geodataframe_equal, assert_geoseries_equal
+
+
```

Comment:
Can you do a global pytest skip if pyarrow is not installed?

(I think ideally the tests should be skipped if the optional dependency is not installed, instead of raising errors)